templates/content/node--oe-publication--teaser.html.twig

```diff
@@ -4,12 +4,16 @@
  * Theme override to display a node of bundle publication in the teaser view mode.
  */
 #}
+{% set image_url = content.oe_publication_thumbnail|field_value|render %}
 {{ pattern('list_item', {
-  'variant': 'thumbnail_primary',
+  'variant': 'thumbnail_secondary',
   'url': url,
   'title': content.extra_field_oe_theme_helper_short_title_with_fallback|field_value,
   'meta': [
-    content.oe_publication_date|field_value
+    content.oe_publication_type|field_value|safe_join(' | '),
+    content.oe_publication_date|field_value,
+    content.oe_author|field_value|safe_join(' | ')
```

Comment:
This was correct as it was before. With @kugta  we had agreed to use commas to separate multiple values of the same type, and pipes for things of different types.